recipes/soxr/bld.bat

```diff
@@ -0,0 +1,22 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "NMake Makefiles" ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DBUILD_TESTING=ON ^
+    %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build.
+cmake --build . --config Release
+if errorlevel 1 exit 1
+
+:: Install.
+cmake --build . --config Release --target install
+if errorlevel 1 exit 1
+
+:: Test.
+ctest --output-on-failure -C Release -E "misc::check_license"
```

Comment:
Unless there is a `misc::check_license` test, it does not make sense to try to skip a test name like that (the `-E` option of ctest just excluds the test that contain `misc::check_license` in their name).

Reply:
sorry I copied the file from another receipt..

There's no test with this name, I'll remove it!